lib/mix/lib/mix/tasks/release.ex

```diff
@@ -492,7 +492,9 @@ defmodule Mix.Tasks.Release do
   can pass anonymous functions before and after the `:assemble` to
   customize your release assembling pipeline. Those anonymous functions
   will receive a `Mix.Release` struct and must return the same or
-  an updated `Mix.Release` struct.
+  an updated `Mix.Release` struct. It is also possible to build a tarball
+  of the release by passing the `:tar` step anywhere after `:assemble`.
+  The tarball is created in `_build/MIX_ENV/rel/RELEASE_NAME/releases/RELEASE_VSN/RELEASE_NAME.tar.gz`
```

Comment:
I would like to propose to put the tar at: `_build/MIX_ENV/rel/RELEASE_NAME-RELEASE_VSN.tar.gz`

I don't want to put it inside the release because it increases the odds of someone accidentalaly zipping it again  later and having more stuff to pass around.